addons/hw_drivers/main.py

```diff
@@ -92,11 +92,17 @@ def send_alldevices(self, iot_client=None):
             _logger.info('Ignoring sending the devices to the database: no associated database')
 
     def run(self):
-        """
-        Thread that will load interfaces and drivers and contact the odoo server with the updates
-        """
+        """Thread that will load interfaces and drivers and contact the odoo server with the updates"""
         self.server_url = helpers.get_odoo_server_url()
         helpers.start_nginx_server()
+        # Reset log level to default if the time has come
+        log_level_reset_timestamp = helpers.get_conf('log_level_reset_timestamp')
+        if log_level_reset_timestamp and log_level_reset_timestamp <= time.time():
+            helpers.update_conf({
+                'log_level_reset_timestamp': '',
+                'log_handler': ':WARNING',
+                'log_level': 'warn',
+            })
```

Comment:
If I am not mistaken, this is called only once on boot, is it something intended ? Given the commit message, I thought it would be removed automatically after 1 week without the need to reboot

Reply:
You're right, it would be more efficient if people never reboot their IoT Boxes